<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insertQMessage_Dialog</class>
 <widget class="QDialog" name="insertQMessage_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="insertQMessage_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="comfirm_pushButton">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>150</y>
     <width>123</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>是</string>
   </property>
  </widget>
  <widget class="QPushButton" name="negative_pushButton">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>150</y>
     <width>123</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>否</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/backGround.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>insertQMessage_label</zorder>
  <zorder>comfirm_pushButton</zorder>
  <zorder>negative_pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
